sys/socket_unix.go

```diff
@@ -23,11 +23,16 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/pkg/errors"
 	"golang.org/x/sys/unix"
 )
 
 // CreateUnixSocket creates a unix socket and returns the listener
 func CreateUnixSocket(path string) (net.Listener, error) {
+	// BSDs have a 104 limit
+	if len(path) > 104 {
+		return nil, errors.Errorf("%q: unix socket path too long (> 106)", path)
```

Comment:
Should that be 104 in the message to match the condition?

Reply:
yes, it was fixed in a subsequent PR (https://github.com/containerd/containerd/pull/2258/files)

Reply:
Awesome. Thank you for following up!